File: agencedevoyageBack/src/main/java/com/ppe/entites/Voiture.java
package com.ppe.entites;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Lob;
import javax.persistence.ManyToOne;
import javax.validation.constraints.NotNull;

@Entity
public class Voiture  {
	
	private static final long serialVersionUID = 1L;
	@Id
	@GeneratedValue
	private Long Matricule;
	private String Marque;
	private String Type;
	private int Capacite;
	    private String image;
	
	
	public Voiture() {
	}
	public Voiture(Long matricule, String marque, String type, int capacite, String image) {
		super();
		Matricule = matricule;
		Marque = marque;
		Type = type;
		Capacite = capacite;
		this.image = image;
	}
	public Long getMatricule() {
		return Matricule;
	}
	public void setMatricule(Long matricule) {
		Matricule = matricule;
	}
	public String getMarque() {
		return Marque;
	}
	public void setMarque(String marque) {
		Marque = marque;
	}
	public String getType() {
		return Type;
	}
	public void setType(String type) {
		Type = type;
	}
	public int getCapacite() {
		return Capacite;
	}
	public void setCapacite(int capacite) {
		Capacite = capacite;
	}
	public String getImage() {
		return image;
	}
	public void setImage(String image) {
		this.image = image;
	}
	public static long getSerialversionuid() {
		return serialVersionUID;
	}
	
	

	
	
	
	
	

	

	
	
	
	

}

File: agenceDeVoyageFront/src/app/models/voiture.ts
export class Voiture {
    private _Matricule: number;
    public get Matricule(): number {
        return this._Matricule;
    }
    public set Matricule(value: number) {
        this._Matricule = value;
    }
	private _Marque: String;
    public get Marque(): String {
        return this._Marque;
    }
    public set Marque(value: String) {
        this._Marque = value;
    }
	private _Type: String;
    public get Type(): String {
        return this._Type;
    }
    public set Type(value: String) {
        this._Type = value;
    }
	private _Capacite: number;
    public get Capacite(): number {
        return this._Capacite;
    }
    public set Capacite(value: number) {
        this._Capacite = value;
    }
    private _image: String;
    public get image(): String {
        return this._image;
    }
    public set image(value: String) {
        this._image = value;
    }
    public toString(){
        return this._Capacite;
    }
     constructor(Matricule:number,Marque:String,Type:String,Capacite:number,image:String){
        this.Marque=Marque;
        this.Matricule=Matricule;
        this.Type=Type;
        this.Capacite=Capacite;
        this.image=image;
    } 
}
